File: Go/FizzBuzz.go
// Solution of FizzBuzz in golang 
// author: @divyanshugit
package main
import "fmt"
func main() {
    for i := 1; i <= 100; i++ {

        if i%3 == 0 {
            // Multiple of 3
            fmt.Printf("fizz")
        }
        if i%5 == 0 {
            // Multiple of 5
            fmt.Printf("buzz")
        }

        if i%3 != 0 && i%5 != 0 {
            // Neither, so print the number itself
            fmt.Printf("%d", i)
        }

        // A trailing new line (so both fizz + buzz can be printed on the same line)
        fmt.Printf("\n")

    }
}

File: php/fizzbuzz.php
<?php 
// PHP program to print Fizz Buzz 

$i; 
for ($i = 1; $i <= 100; $i++) 
{ 
	// number divisible by 3 and 
	// 5 will always be divisible 
	// by 15, print 'FizzBuzz' in 
	// place of the number 
	if ($i % 15 == 0) 
		echo "FizzBuzz" . " "; 
	
	// number divisible by 3? print 
	// 'Fizz' in place of the number 
	else if (($i % 3) == 0) 
		echo "Fizz" . " ";			 
	
	// number divisible by 5, print 
	// 'Buzz' in place of the number 
	else if (($i % 5) == 0)				 
		echo "Buzz" . " ";			 

	else // print the number		 
		echo $i," " ;			 
} 


?> 
